test/data/psst-component-data/psst.json

```diff
@@ -1,12 +1,35 @@
 [
     {
+        "name": "a",
         "include": [
             "https://a.com/*"
         ],
         "exclude": [
         ],
         "version": 1,
-        "test_script": "a/test.js",
-        "policy_script": "a/policy.js"
+        "user_script": "user.js",
+        "policy_script": "policy.js"
+    },
+    {
+        "name": "b",
+        "include": [
+            "https://b.com/*"
+        ],
+        "exclude": [
+        ],
+        "version": 1,
+        "user_script": "user.js",
```

Comment:
all of these test rules have the same user script and policy script? That seems problematic to me in terms of testing